src/redux/reducer.js

```diff
@@ -1057,6 +1061,8 @@ function getEquipmentTypeFromUpdateType(updateType) {
     switch (updateType) {
         case 'lines':
             return EQUIPMENT_TYPES.LINE;
+        case 'tieLines':
```

Comment:
What does this update type String refer to? To some data update type somewhere in the back? Shouldn’t it be referenced/commented somewhere?